axum-extra/Cargo.toml

```diff
@@ -84,6 +86,7 @@ allowed = [
     "cookie",
     "futures_core",
     "futures_util",
+    "headers_core",
```

Comment:
Why only the core crate? The entire `headers` crate is being re-exported.

Reply:
That is fixed now.